recipes/zstash/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "zstash" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/zstash/archive/v{{ version }}.tar.gz
+  sha256: 16d4169dcf49057ac86fb78410bffbc9ceba7680f73d4f88659992c18376cb63
+  patches: 
+    - patches/fix-setup-py.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
+    - six
+    - globus-sdk >=2
```

We likely want
```suggestion
    - globus-sdk >=3.0.0,<4.0.0
```
But let's see what happens upstream before we make the change.